File: NET/com.yupanastudio.igest.winapp.model/admin/Sistema.cs
using System.Collections.Generic;

namespace com.yupanastudio.igest.winapp
{
    public class Sistema
    {
        public string idSistema { get; set; }
        public string nomSistema { get; set; }
        public string nomCorto { get; set; }
        public string descripcion { get; set; }
        public int orden { get; set; }
        public List<Opcion> lsOpcionX { get; set; }

    }//end class

}

File: SPRING/igest.server/src/main/java/com/yupanastudio/igest/server/principal/model/Sistema.java
/*
 * Document    : Sistema
 * Author      : Franklin Barzola Olivares <frankoantonio@gmail.com>
 */
package com.yupanastudio.igest.server.principal.model;

import org.springframework.data.annotation.Id;

public class Sistema {

    @Id
    private String idSistema;
    private String nomSistema;
    private String nomCorto;
    private String descripcion;
    private int codigo;

    public String getIdSistema() {
        return idSistema;
    }

    public void setIdSistema(String idSistema) {
        this.idSistema = idSistema;
    }

    public String getNomSistema() {
        return nomSistema;
    }

    public void setNomSistema(String nomSistema) {
        this.nomSistema = nomSistema;
    }

    public String getNomCorto() {
        return nomCorto;
    }

    public void setNomCorto(String nomCorto) {
        this.nomCorto = nomCorto;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public int getCodigo() {
        return codigo;
    }

    public void setCodigo(int codigo) {
        this.codigo = codigo;
    }

}
